jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ConnectionMetaData.java

```diff
@@ -52,7 +52,9 @@ default boolean isSecure()
 
     /**
      * @return whether the functionality of pushing resources is supported
+     * @deprecated in favour of 103 Early Hints
      */
+    @Deprecated
```

Comment:
Orthogonal change?

Reply:
yes sorry.  We have not been reviewing 12.1 changes unless necessary. I thought the redirect changes needed review, but some other servlet 6.1 changes got swept up.